ui/src/views/scenes/Filters.vue

```diff
@@ -20,7 +20,8 @@
             <option value="rating_asc">↑ {{$t("Rating")}}</option>
             <option value="scene_added_desc">↓ {{$t("Scene added date")}}</option>
             <option value="scene_updated_desc">↓ {{$t("Scene updated date")}}</option>
-            <option value="last_opened">↻ {{$t("Recently viewed")}}</option>
+            <option value="last_opened">↓ {{$t("Last viewed date")}}</option>
```

Comment:
Since you're adding a second sort order, you should keep with the pattern and add the `_desc` suffix here.